src/main/java/org/takes/rq/multipart/RqMtBase.java

```diff
@@ -183,8 +186,9 @@ public InputStream body() throws IOException {
     private Map<String, List<Request>> requests(
         final Request req) throws IOException {
         final String header = new RqHeaders.Smart(req).single("Content-Type");
-        if (!new EnglishLowerCase(header).string()
-            .startsWith("multipart/form-data")) {
+        if (!new UncheckedText(
+            new Lowered(header)
+        ).asString().startsWith("multipart/form-data")) {
```

Comment:
@fanifieiev please, leave a puzzle for [it](https://github.com/yegor256/cactoos/issues/1265).